docs/advanced_usage.rst

```diff
@@ -841,6 +841,31 @@ return books for the publisher::
         class Meta:
             model = Book
 
+.. _interoperability:
+
+Interoperability with 3rd party libraries
+-----------------------------------------
+
+import_export extends the Django Admin interface.  There is a possibility that clashes may occur with other 3rd party
+libraries which also use the admin interface.
+
+django-admin-sortable2
+^^^^^^^^^^^^^^^^^^^^^^
+
+Issues have been raised due to conflicts with setting `change_list_template <https://docs.djangoproject.com/en/dev/ref/contrib/admin/#django.contrib.admin.ModelAdmin.change_list_template>`_.  There is a workaround listed `here <https://github.com/jrief/django-admin-sortable2/issues/345#issuecomment-1680271337>`_.
```

Comment:
I think it's better to use the stable version link:
https://docs.djangoproject.com/en/stable/ref/contrib/admin/#django.contrib.admin.ModelAdmin.change_list_template

Reply:
Done

Reply:
The update commit is yet to come

Reply:
Apologies - is there now